WorldProcessor/WorldProcessor.Application/Handlers/WorldProcessing/Dto/TileConfigDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace WorldProcessor.Application.Handlers.WorldProcessing.Dto
+{
+    public class TileConfigDto
+    {
+        public string Id { get; set; }
+
+        public NeighborConstraintDto[]? Neighbors { get; set; }
```

Comment:
This was made due to test world config wasn't work, because "forest" and "field" don't have neighbors property.
It can be solved by initializing this property in constructor, but i think that will be a logic, that dto should not contain by definition. What you think, @Max-Dov?